azure-pipelines.yml

```diff
@@ -2,7 +2,7 @@
 trigger:
 - master
 
-# Branches that trigger builds on PR
+# Branch(es) that trigger build(s) on PR
```

Comment:
If we're being optionally plural it's  "Branch(es) that trigger(s) build(s) on PR" :)